src/Elcodi/Bundle/AttributeBundle/DependencyInjection/ElcodiAttributeExtension.php

```diff
@@ -77,15 +77,15 @@ protected function getConfigurationInstance()
     protected function getParametrizationValues(array $config)
     {
         return [
-            "elcodi.core.attribute.entity.attribute.class" => $config['mapping']['attribute']['class'],
-            "elcodi.core.attribute.entity.attribute.mapping_file" => $config['mapping']['attribute']['mapping_file'],
-            "elcodi.core.attribute.entity.attribute.manager" => $config['mapping']['attribute']['manager'],
-            "elcodi.core.attribute.entity.attribute.enabled" => $config['mapping']['attribute']['enabled'],
+            "elcodi.entity.attribute.class" => $config['mapping']['attribute']['class'],
```

Comment:
I find it confusing to call a parameter `['mapping']['attribute']['class']` in PHP and `entity.attribute.class` as parameter.  `entity` and `mapping` are not the same concepts.